packages/kit/types/ambient.d.ts

```diff
@@ -113,10 +113,22 @@ declare module '$app/navigation' {
 		opts?: { replaceState?: boolean; noscroll?: boolean; keepfocus?: boolean; state?: any }
 	): Promise<void>;
 	/**
-	 * Causes any `load` functions belonging to the currently active page to re-run if they `fetch` the resource in question. If no argument is given, all resources will be invalidated. Returns a `Promise` that resolves when the page is subsequently updated.
+	 * Causes any `load` functions belonging to the currently active page to re-run if they registered it with `fetch` or `depends`. Returns a `Promise` that resolves when the page is subsequently updated.
+	 *
+	 * If no argument is given, all resources will be invalidated.
```

Comment:
Separate to this PR but I think we should probably create an `invalidateAll()` function — makes the `invalidate()` interface a bit less cluttered, and will be beneficial if we need to add an options object (since `invalidate(undefined, options)` is weird)

Reply:
implemented in #6493